src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/RedundantModifierCheck.java

```diff
@@ -58,12 +59,12 @@
  * </li>
  * </ol>
  * <p>
- * Interfaces by definition are abstract so the {@code abstract}
- * modifier on the interface is redundant.
+ * {@code interface} and {@code @interface} by definition are abstract so the {@code abstract}
+ * modifier is redundant on them.
  * </p>
- * <p>Classes inside of interfaces by definition are public and static,
- * so the {@code public} and {@code static} modifiers
- * on the inner classes are redundant. On the other hand, classes
+ * <p>Type declarations nested under an interface and an annotated interface by definition
```

Comment:
`annotated interface` please change it to `{@code interface} and {@code @interface}` as above

Reply:
Done